apps/condo/domains/miniapp/utils/b2bAppServiceUserAccess/server.utils.js

```diff
@@ -132,6 +132,41 @@ const canManageByServiceUser = async ({ authentication: { item: user }, listKey,
     return !isEmpty(B2BAppContexts)
 }
 
+const canExecuteByServiceUser = async (args, schemaConfig) => {
+    const { authentication: { item: user }, args: { data }, info: { fieldName } } = args
```

Comment:
You can use gqlName here

Reply:
https://github.com/open-condo-software/condo/blob/master/apps/condo/domains/user/access/RegisterNewServiceUserService.js#L8

Reply:
Yes, refactored